apps/02RockPaperScissors.js

```diff
@@ -10,8 +10,30 @@ var rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  if (hand1.toLowerCase() === hand2.toLowerCase()) {
```

Comment:
You could simplify your code by doing the '.toLowerCase()' outside of the if/else statements.
So directly after your function begins, you would put:
hand1 = hand1.toLowerCase();
hand2 = hand2.toLowerCase();